airbyte-db/db-lib/src/main/java/io/airbyte/db/SourceOperations.java

```diff
@@ -12,8 +12,5 @@
 
   JsonNode rowToJson(QueryResult queryResult) throws SQLException;
 
-  JsonSchemaType getJsonType(SourceType sourceType);
-
-  //
-  // JsonSchemaType getJsonSchemaType(SourceType columnType);
+  JsonSchemaType getAirbyteType(SourceType sourceType);
```

Comment:
Seems self-explanatory, can we add a Javadoc link to where the Airbyte types are defined? 

```java
Retrieves associated {@link JsonSchemaType.java}
```

Reply:
Updated comment